app/code/Magento/Dhl/Model/Carrier.php

```diff
@@ -994,7 +994,7 @@ function (array $a, array $b): int {
                 $code = $bodyXml->xpath('//GetQuoteResponse/Note/Condition/ConditionCode');
                 if (isset($code[0]) && (int)$code[0] == self::CONDITION_CODE_SERVICE_DATE_UNAVAILABLE) {
                     $debugPoint['info'] = sprintf(
-                        __("DHL service is not available at %s date"),
+                        __("DHL service is not available at %s date")->render(),
                         $responseData['date']
                     );
```

Comment:
I see 2 possible solutions here:
1. skip `Phrase` usage, because it's log data and in my opinion log data should not be translated
```
$debugPoint['info'] = sprintf("DHL service is not available at %s date", $responseData['date']);
```
2. skip `sprintf` usage
```
$debugPoint['info'] = __("DHL service is not available at %1 date", $responseData['date']);
```

1 is more preferable for me.

Reply:
None of the suggested solutions aren't suitable:
1. Why log data shouldn't be translated? Changing the existing behaviour, isn't what we're looking for.
2. This is BIC. This was already discussed internally, and it was decided to use the existing implementation.